packages/agents-sdk/src/agent.ts

```diff
@@ -1119,234 +1119,6 @@ export class Agent implements AgentInterface {
     }
     return [input];
   }
-
-  private async saveToDatabase(): Promise<void> {
```

Comment:
@miles-kt-inkeep @amikofalvy I think it's safe to remove these unused `private` methods?